config.js

```diff
@@ -3,24 +3,51 @@ import cors from 'cors'
 import helmet from 'helmet'
 import morgan from 'morgan'
 import bodyParser from 'body-parser'
+import schedule from "node-schedule"
+import nodemailer from "nodemailer"
+
+let transporter; //nodemailer
+
+import { roleNotifier } from "./jobs";
 
 const setHeaders = res => {
     res.setHeader('Access-Control-Allow-Origin', '*');
     res.setHeader('Access-Control-Allow-Methods', 'GET, POST, PUT, DELETE');
     res.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,content-type');
     res.setHeader('Access-Control-Allow-Credentials', true);
-}
+};
 
 const mongo_connect = () => {
     mongoose.set('useCreateIndex', true);
-    mongoose.connect(process.env.MONGO_URL, { useNewUrlParser: true })
-}
+    mongoose.connect(process.env.MONGO_URL, { useNewUrlParser: true });
+};
 
 const useMiddleware = app => {
-    app.use(cors())
-    app.use(helmet())
-    app.use(morgan('dev'))
-    app.use(bodyParser.json())
-}
+    app.use(cors());
+    app.use(helmet());
+    app.use(morgan('dev'));
+    app.use(bodyParser.json());
+};
+
+const initNodeMailer = () => {
+
+    const { MAIL_SERVICE, MAIL_EMAIL, MAIL_PASSWORD, MAIL_HOST } = process.env;

```

Comment:
Could it be possible to store it in one object MAIL?
`MAIL.service
MAIL.username
MAIL.password
MAIL.host`

Reply:
It not only could, it definetely should. But where should we create this object?

Reply:
After deeper investigation. It is possible only if we store stringified JSON
https://stackoverflow.com/questions/34418536/can-i-store-objects-in-process-env